front-end/src/components/Reactionbar/ReactionButton.tsx

```diff
@@ -32,17 +33,17 @@ const ReactionButton = function ({
 	className,
 	reaction,
 	count,
-	userIds,
+	userNames,
 	postId,
 	commentId,
 	refetch
 }: ReactionButtonProps) {
-	const { id } = useContext(UserDetailsContext);
+	const { id, username } = useContext(UserDetailsContext);
```

Comment:
nit we can remove the `id` here, it is used in an `if` bellow, but we can use the username here as well rather than the id.

Reply:
id used in addPostReactionMutation, deletePostReactionMutation can't remove